beacon_node/src/config.rs

```diff
@@ -245,15 +249,22 @@ pub fn get_config<E: EthSpec>(
     if cli_args.is_present("merge") || cli_args.is_present("execution-endpoints") {
         let mut el_config = execution_layer::Config::default();
 
-        if let Some(endpoints) = cli_args.value_of("execution-endpoints") {
+        if let Some(endpoints) = cli_args.value_of("execution-endpoint") {
             client_config.sync_eth1_chain = true;
-            el_config.execution_endpoints = endpoints
+            let mut execution_endpoints: Vec<SensitiveUrl> = endpoints
                 .split(',')
                 .map(SensitiveUrl::parse)
                 .collect::<Result<_, _>>()
                 .map_err(|e| format!("execution-endpoints contains an invalid URL {:?}", e))?;
+            // Retain only first element since we don't support multiple execution engines
+            execution_endpoints.truncate(1);
+            el_config.execution_endpoints = execution_endpoints;
         } else if cli_args.is_present("merge") {
-            el_config.execution_endpoints = client_config.eth1.endpoints.clone();
+            el_config.execution_endpoints =
+                vec![
+                    SensitiveUrl::parse(execution_layer::DEFAULT_EXECUTION_ENDPOINT)
```

Comment:
If we set it as a clap default we won't have a way for users to "opt-in" to the merge. It would mean that present-day mainnet users would start getting logs saying that their current version of Geth/Nethermind/etc isn't responding on the authenticated engine API port.